app/src/main/java/com/apator/map/helpers/mappers/SolarDetailsJSONToDb.kt

```diff
@@ -0,0 +1,20 @@
+package com.apator.map.helpers.mappers
+
+import com.apator.map.database.Entity.DetailsEntity
+import com.apator.map.helpers.formater.DoubleToString
+import com.apator.map.model.singlesolar.Solar
+
+object SolarDetailsJSONToDb {
+    fun map(solar: Solar) = DetailsEntity(
+        solar.inputs!!.file_id!!,
+        solar.stationInfo!!.elev!!,
+        solar.stationInfo.tz!!,
+        solar.stationInfo.location!!,
+        solar.stationInfo.city!!,
+        solar.stationInfo.state!!,
+        DoubleToString.format(';',solar.outputs!!.acMonthly),
```

Comment:
separator do stałej i to nawet powinien być w serializerze. Raczej nie ma potrzeby żeby programista zmieniał operator.